mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/MovableChessBoardState.kt

```diff
@@ -0,0 +1,39 @@
+package ch.epfl.sdp.mobile.ui.game
+
+import androidx.compose.runtime.Stable
+
+/**
+ * An interface that extend the interface [ChessBoardState] and offers some ways for the
```

Comment:
```suggestion
 * An interface that extends the interface [ChessBoardState] and offers some ways for the
```

Reply:
Fix in commit https://github.com/epfl-SDP/android/pull/240/commits/23fd568305d5eeeb1ea1c92c3947b89dcb54407a